tutor/specs/e2e/assignment-review.e2e.ts

```diff
@@ -1,12 +1,24 @@
+import { DateTime } from 'luxon'
 import { visitPage, withUser, test, faker, expect } from './test'
 const COURSE_ID = 1
 const HW = 4
 const RD = 2
 const GRADABLE=3
 
-test.describe('Assignment Review', () => {
+const openCalendarButtonCSS = '.oxdt-dropdown:not(.oxdt-dropdown-hidden) button'
+const openCalendarCellTodayCSS = '.oxdt-dropdown:not(.oxdt-dropdown-hidden) .oxdt-cell-today'
+const openCalendarCellAfterSelectedCellCSS = '.oxdt-dropdown:not(.oxdt-dropdown-hidden) ' +
+                                             '.oxdt-cell-selected + .oxdt-cell'
+
+const detailsDueAtCSS = '.tasking-date-time.row + .tasking-date-time.row .due-at'
+const detailsClosesAtCSS = '.tasking-date-time.row + .tasking-date-time.row .closes-at'
```

Comment:
typically in tests it's best to keep the selectors as generic as possible so they don't break if the DOM's rearranged.  that's the rational behind adding the data-test-id attributes.  Might be better to use those if possible here